css/templates/land-see/page.css

```diff
@@ -279,3 +279,89 @@ h4,
     justify-content: flex-end;
   }
 }
+
+/* Subscribe */
+
+.land-see-subscribe {
+  position: relative;
+  height: 310px;
+  overflow: hidden;
+  color: var(--color-font-footer);
+}
+
+.land-see-subscribe amp-img {
+  position: absolute;
+  width: 100%;
+  object-position: top center;
+}
+
+amp-img.cover img {
+  object-fit: cover;
+}
+
+.land-see-subscribe-bg {
+  z-index: 1;
+}
+
+.land-see-subscribe-form {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 2;
+}
+
+.land-see-subscribe-form form {
+  padding: 0;
+}
+
+.land-see-subscribe-form h2 {
+  margin-bottom: var(--h3);
+  padding-top: var(--h1);
+}
+
+.land-see-subs-heading-text {
+  font-size: var(--h2);
+  position: relative;
+}
+
+.land-see-subs-heading-text:after {
+  content: 'Land+See';
```

Comment:
I'm attempting to avoid having to modify the component or feature multiple header tags to get the bold effect in the design.  I'm using the base form component and passing the previous part of the sentence "Weekly inspiration from " through JSON.

```
    "form": {
      "title": "Subscribe to Land + See",
      "method": "GET",
      "action": "/",
      "heading": {
        "level": 2,
        "classes": "land-see-subs-heading-text",
        "text": "Weekly inspiration from "
      }
```

How would you recommend featuring just the last couple of words in bold using that form component? Styling multiple header tags doesn't seem like a good solution, either. Example:

```
      "heading": {
        "level": [
            {
                2, "classes": "land-see-subs-heading-text", "text": "Weekly inspiration from "
            },
            {
                3, "classes": "land-see-subs-heading-text-bold", "text": "Land+See"
            }
      }
```

Speaking of... I just noticed that the default on that `form.snip.html` component IS an `h2`:

```
      {{^level}}
        <h2 class="{{classes}}">{{text}}</h2>
      {{/level}}
```

Reply:
You can change the text to allow html (just for cases like these)
```
{{{text}}}
```
would solve this issue and you can pass in a html with a span.

Reply:
Perfect! Will get that out later this afternoon/evening.